build/configs/artik053/tools/openocd/partition_gen.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+###########################################################################
+#
+# Copyright 2016-2017 Samsung Electronics All Rights Reserved.
```

Comment:
just cross check, it may have to be just 2017

Reply:
Ok, I will check on this.